app/core/tests/test_admin.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase, Client
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+
+class AdminSiteTests(TestCase):
+
+    def setUp(self):
```

Comment:
Do you skip `super().setUp()` on purpose?

Reply:
nope,